migration/identity-migration/src/test/java/io/camunda/migration/identity/TenantMappingRuleMigrationHandlerTest.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migration.identity;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import io.camunda.migration.identity.dto.MappingRule.Operator;
+import io.camunda.migration.identity.dto.MigrationStatusUpdateRequest;
+import io.camunda.migration.identity.dto.Tenant;
+import io.camunda.migration.identity.dto.TenantMappingRule;
+import io.camunda.migration.identity.midentity.ManagementIdentityClient;
+import io.camunda.migration.identity.midentity.ManagementIdentityTransformer;
+import io.camunda.migration.identity.service.MappingService;
+import io.camunda.migration.identity.service.TenantService;
+import io.camunda.search.entities.TenantEntity;
+import io.camunda.zeebe.client.api.command.ProblemException;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+public class TenantMappingRuleMigrationHandlerTest {
+  final ArgumentCaptor<Collection<MigrationStatusUpdateRequest>> migrationStatusCaptor =
+      ArgumentCaptor.forClass(Collection.class);
+  @Mock private ManagementIdentityClient managementIdentityClient;
+  @Mock private TenantService tenantService;
+  @Mock private MappingService mappingService;
+
+  private final ManagementIdentityTransformer managementIdentityTransformer =
+      new ManagementIdentityTransformer();
+  private TenantMappingRuleMigrationHandler migrationHandler;
+
+  @BeforeEach
+  void setUp() throws Exception {
+    MockitoAnnotations.openMocks(this).close();
```

Comment:
:question: What is this doing?

Reply:
It's for creating @Mock instances. If you know a better approach, let me know and I apply in next tests.

Reply:
Ah okay, I think you could also just annotate the test class with ` @ExtendWith(MockitoExtension. class)`